saturn-r/postBuild

```diff
@@ -0,0 +1,48 @@
+# https://cran.rstudio.com/bin/linux/debian/
+sudo apt-key adv --keyserver keys.gnupg.net --recv-key 'E19F5F87128899B192B1A2C2AD5F960A256A04AF'
+
+sudo apt-get update
+sudo apt-get install -y \
+    --no-install-recommends \
+        software-properties-common
+
+sudo su -c \
+    "echo 'deb http://cloud.r-project.org/bin/linux/debian buster-cran40/' >>/etc/apt/sources.list"
+
+sudo apt update
+sudo apt install -y -t buster-cran40 \
+    r-base \
+    r-base-dev 
+
+sudo apt-get update
+sudo apt-get install -y \
+    jupyter-client \
+    devscripts \
+    libxml2 \
+    libxml2-dev \
+    libcurl4-openssl-dev \
+    libssl-dev \
+    libzmq3-dev \
+    qpdf
+
+# make it possible to `install.packages()` - write permissions required.
+sudo chown 1000:1000 -R $R_LIBS
+sudo chmod 777 -R $R_LIBS
+
+# install other packages
+sudo Rscript -e "install.packages(c( \
+        'caret', \
+        'data.table', \
+        'dplyr', \
+        'ggplot2', \
+        'IRkernel', \
+        'lightgbm', \
+        'Rcpp', \
+        'remotes', \
+        'reticulate', \
+        'stringr', \
+        'tidyr', \
+        'xgboost' \
+    ), \
+    dependencies = c('LinkingTo', 'Depends', 'Imports') \
+    )"
```

Comment:
I learned that you can tell install.packages to run in parallel with the `Ncpus, it might be worth doing here to make the installations go faster:

https://www.r-craft.org/r-news/faster-r-package-installation/

Reply:
Would the behavior of this depend on what kind of machines the user has requested?

Reply:
the type of machine users choose use shouldn't matter since here you're only talking about what we do at the moment this image is built, which happens upstream of anyone pulling it to use in Saturn

Reply:
Added!

Reply:
Ncpus is a nice trick! I didn't know about that one. Thank you!